src/index.js

```diff
@@ -225,6 +225,11 @@ export default class AuthnWidget {
       });
     }
 
+    let firstTextInput = document.querySelector("input[type=text]:not(:disabled)");
```

Comment:
Do you think we should include date, email, and phone as field types to focus as well?

Reply:
We might want to clean up all the `autofocus` references in the templates where it makes sense.

Reply:
hmm, I didn't notice autofocus was placed in some of the other partials already, perhaps it's better to place this in a post render callback for just registration instead to keep that level of control.

Reply:
@khosravian What do you think about this? Should we have global autofocus or leave it up to the templates to determine which field gets the focus?

Reply:
@mketchum I think we need both, global autofocus on the first field and a way to override the behavior at the template level.

Reply:
global autofocus first input (date, email, phone, pass, text) unless autofocus specified in template